tests/mock_tests/check.h

```diff
@@ -4,7 +4,15 @@
 #include <iostream>
 #include <vector>
 
-#include "saiattributelist.h"
+#include "SaiAttributeList.h"
+
+#include <hiredis/hiredis.h>
+#include "swss/dbconnector.h"
+#include "swss/logger.h"
+#include "sai_serialize.h"
+#include "string.h"
+
+using namespace saimeta;
```

Comment:
Not 'using namespace' in a header file

Reply:
i know, but that was in header previously as legacy, i can remove this but it will require to fix in many places probably